### PR TITLE
Run3-sim44 Make changes to conform G4/GV Comparison

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Make option for registering the EM parametrization process and add a region for NoSD option. These are useful for Geant4/GeantV comparisons

#### PR validation:

Tested with standard run flow matrix

#### if this PR is a backport please specify the original PR:

Nothing special

### CODE REVIEW

File: SimG4Core/Application/python/g4SimHits_cfi.py [hunk 1]
```diff
@@ -50,6 +50,7 @@
     PhysicsTablesDirectory = cms.untracked.string('PhysicsTables'),
     StorePhysicsTables = cms.untracked.bool(False),
     RestorePhysicsTables = cms.untracked.bool(False),
+    UseParametrisedEMPhysics = cms.untracked.bool(False),
```
Comment (kpedro88) [inline SimG4Core/Application/python/g4SimHits_cfi.py]: should this be true by default to keep previous behavior?

Comment (bsunanda) [reply SimG4Core/Application/python/g4SimHits_cfi.py]: You are right - corrected

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by kpedro88** [comment]
+upgrade

**Comment by cvuosalo** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**